homework/index.js

```diff
@@ -30,13 +30,93 @@
     return elem;
   }
 
+  function renderRepository(leftContainer, repository) {
+    const tabel = createAndAppend('tabel', leftContainer, { id: 'tabel' });
+    const tr1 = createAndAppend('tr', tabel, { class: 'descriptions' });
+
+    createAndAppend('td', tr1, { class: 'col1_data', text: 'Repository:' });
+    createAndAppend('td', tr1, { class: 'col2_data', text: repository.name });
+
+    const tr2 = createAndAppend('tr', tabel, { class: 'descriptions' });
+
+    createAndAppend('td', tr2, { class: 'col1_data', text: 'Description:' });
+    createAndAppend('td', tr2, { class: 'col2_data', text: repository.description });
+
+    const tr3 = createAndAppend('tr', tabel, { class: 'descriptions' });
+
+    createAndAppend('td', tr3, { class: 'col1_data', text: 'Forks:' });
+    createAndAppend('td', tr3, { class: 'col2_data', text: repository.forks });
+
+    const tr4 = createAndAppend('tr', tabel, { class: 'descriptions' });
+
+    createAndAppend('td', tr4, { class: 'col1_data', text: 'Updated:' });
+    createAndAppend('td', tr4, { class: 'col2_data', text: repository.updated_at });
```

Comment:
This doesn't seem to be a requirement, but as an improvement, there is a way to format the date you're getting from the API and make it more human readable, by using this method on new Date(repository.updated_at): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString